<launch>
    <arg name="transformed_cloud" default="base_frame_point_cloud" />
    <arg name="compute_bounding_boxes" default="False" />

    <arg name="source_cloud" default="/head_camera/depth_registered/points" />
    <arg name="transform_from" default="head_camera_rgb_optical_frame" />
    <arg name="transform_to" default="base_link" />

    <node name="segment_point_cloud_publisher" pkg="fetch_vr" type="segment_point_cloud_publisher.py">
        <param name="input_cloud_topic" value="$(arg transformed_cloud)" />
        <param name="compute_bounding_boxes" value="$(arg compute_bounding_boxes)" />
    </node>

    <node name="transform_point_cloud_publisher" pkg="fetch_vr" type="transform_point_cloud_publisher.py">
        <param name="input_cloud_topic" value="$(arg source_cloud)" />
        <param name="source_frame_topic" value="$(arg transform_from)" />
        <param name="transform_frame_topic" value="$(arg transform_to)" />
    </node>

</launch>
